cloudigrade/api/clouds/azure/tasks/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
+Celery tasks related to Azure for use in the api v2 app.
+
+Related tasks have been grouped into modules to improve readability of the code.
+Only Celery-runnable task functions should be exposed here at the package level.
+
+Important notes for developers:
+
+If you find yourself adding a new Celery task, please be aware of how Celery
+determines which queue to read and write to work on that task. By default,
+Celery tasks will go to a queue named "celery". If you wish to separate a task
+onto a different queue (which may make it easier to see the volume of specific
+waiting tasks), please be sure to update all the relevant configurations to
+use that custom queue. This includes CELERY_TASK_ROUTES in config and the
+Celery worker's --queues argument (see deployment-configs.yaml in shiftigrade
+and related configs in e2e-deploy and saas-templates).
```

Comment:
I figure this was copy-pasted from the other similar task modules. FWIW, the recently updated version of this text in those modules reads:
```suggestion
Celery worker's --queues argument (see related openshift deployment config files
elsewhere and in related repos like e2e-deploy and saas-templates).
```
Some day we should consolidate and clean up these notes, but I think duplicating them like this is fine for now.